### ISSUE TITLE
schedule_cache_dir directory location causes permission errors

### ISSUE THREAD

**Issue by james-vincent** [baker-laboratory/rf_diffusion_all_atom#9]
In the file diffusion.py, in the function def get_chi_betaT  , at line 52, the location of schedule_cache_dir is set to the location of __file__:

    schedule_cache_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'schedule_cache')

This means users must have write permission in the original source directory the programs is launched from. 

This precludes using in a multiuser environment. We install once in a global location for all users and do not give write permission to that location. 

I'm not familiar with running rfdiffusion, but there does not seem to be an option to override this particular cache directory. 

Is there a way to specify another location for this cache?

**Comment by bmilash** [comment]
I agree with james-vincent's comment about caching. Another instance of this is on line 763 of the same file:
     self.cache_dir = os.path.join(os.path.dirname(os.path.abspath(__file    __)), 'cached_schedules') 
It would be great if we could configure the base cache directory, and have it default to $HOME/rf_diffusion_cache or some other location outside the software installation directory. Without this capability it is impossible to create a centrally-installed copy of rf_diffusion.
